### DISCUSSION TITLE
Git Push failing with account suspends fatal error message.

### DISCUSSION THREAD

**Discussion by git-chinmay** [community/community#147622] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### GitHub Feature Area

Other

### Body

I am facing fatal error while doing `git push` . I am able to access all the repos from github website but whenever making any code changes and trying to do git push its giving below error.
`remote: Your account is suspended. Please visit https://support.github.com for more information.
fatal: unable to access 'https://github.com/git-chinmay/terraform.git/': The requested URL returned error: 403`

I have opened a support ticket 3104368 but its been a month now and I have not receiving any response from GitHub support team. Could anybody help me here?

**Comment by zwingthomas** [comment]
It looks like your GitHub account has been flagged or suspended. Unfortunately, only GitHub Support can resolve a suspended account—community members and GitHub Discussions can’t directly lift the suspension. Here are your best steps:

1. **Check Your Email**  
   - GitHub often sends an email to the address on your account explaining why the account was flagged, or asking for additional information. If you haven’t seen such an email, check your spam folder or any alternate email addresses.

2. **Follow Up with GitHub Support**  
   - Since you’ve already opened a support ticket (#3104368) and haven’t heard back for a month, try following up by replying to the existing ticket email. 
   - Additionally, you can submit a feature request or question in the [GitHub Feedback repository](https://github.com/github/feedback/discussions) to inquire about the delay. The main resolution, though, has to come from the GitHub Support team.

3. **Review Terms and Content**  
   - Sometimes an account is flagged if there’s a suspicion of violating [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service) or [Community Guidelines](https://docs.github.com/en/github/site-policy/github-community-guidelines). Double-check that your repositories, content, or activity comply.

4. **Note: You cannot use a Different Account**  
   - Users may only have one free account per person. You may be able to make a paid account. I am not sure.

Only GitHub can reinstate your account. Keep reaching out to Support with the ticket number, and hopefully they can clarify why your account was suspended and how to restore access. If you’re not receiving email responses, it may help to update your email address on the account or create a new ticket with an alternate email.

I know this is not the answer you want to receive. Please mark this as answered though if you at least found it directionally helpful. Also getting rid of a second GitHub account may be important as that might result in more harsh consequences.

**Comment by zwingthomas** [comment]
*Side Note*

A “suspended” or “flagged” account on GitHub often means **write permissions** are disabled (push, PR merges, etc.), while **read-only access** remains intact. GitHub typically continues to let you:

- **Log in**,  
- **Browse and view repositories** (especially public repos),  
- **Comment or open issues** on certain repos (depending on the suspension level).

They do this so you can still see your code, download it, or clone repos for local reference. However, anything requiring **write** or **admin** access (push, merging PRs, creating new repos) may be blocked until the account is fully reinstated by GitHub Support. So it’s normal to be able to view repos but not push while suspended.
